how-to-pitch/readme.md

```diff
@@ -0,0 +1,49 @@
+> Goal of this guide: Helping you to pitch
+
+# How to pitch
```

Comment:
Wow! Nice! I like. Only thing I think is that there's a lot of boldness going around here - maybe try to highlight only a few words per sentence? And it would be nice to have explanations of certain points on a new line to keep it more visually structured.